gaia/parser.py

```diff
@@ -35,12 +40,29 @@
     gaia.geo.geo_inputs, inspect.isclass) if x[0].endswith('IO')])
 
 
+def load_plugins():
```

Comment:
@aashish24 I moved the function to core.py

My preference would be to keep at minimum the basic vector inputs and processes included in gaia, providing some built-in functionality without requiring the installation of an additional plugin. 

Maybe after this is merged into master we can create a repo for each plugin and move the relevant code over?